recipe/meta.yaml

```diff
@@ -42,6 +45,7 @@ test:
     imports:
         - iris
         - iris.fileformats.pp_packing  # [not win]
+        - iris.fileformats._pyke_rules.compiled_krb  # [not win]
```

Comment:
Yes it must. Thanks for the reminder. The same magic line will be needed in the bld.bat.